src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
 
 export const CuriosityMap: React.FunctionComponent = () => {
+  const [date, setDate] = useState('0')
+
+  function handleClick(day:string) {
+    setDate(day)
+    console.log(date)
+  }
   return (
-    <div>
-        CuriosityMap
-    </div>
+    <main>
```

Comment:
I don't think this should be a `<main>` - `<main>` is semantically used to mark the main content of the webpage, and I think we've been using it at a higher level (i.e. in the `...Page.tsx` components) to hold the whole page. This should be a `<div>`, or possibly even a non-element "fragment": `<>...</>`